tasks/sep/fp-recovery/005-set-game-implementation/justfile

```
@@ -0,0 +1,23 @@
+set positional-arguments
+
+# default recipe to display help information
+default:
+  @just --list
+
+# Generate the `input.json` with the game type and implementation to set.
+generate-input gameType='' implAddr='':
+  #!/usr/bin/env bash
+  set -euo pipefail
+  . .env
+  SET_GAME_TYPE_SIG="setImplementation(uint32, address)"
```

cast does normalize the signature, so this is just a nit to be a bit more defensive against cast bugs
```suggestion
  SET_GAME_TYPE_SIG="setImplementation(uint32,address)"
```